std_msgs/Header header
BoundingBox[] boundingBoxes